framework/tests/viewpointtest/VarargType.java

```diff
@@ -0,0 +1,27 @@
+import viewpointtest.quals.*;
+
+public class VarargType {
+
+    VarargType(String str, Object... args) {}
+
+    void foo() {
+        VarargType a = new VarargType("testStr", new Object());
+    }
+
+    // inner class
+    class Inner {
+        Inner(String str, Object... args) {}
+
+        void foo() {
+            Inner a = new Inner("testStr", new Object());
+        }
+    }
+
+    // anonymous class
+    Object o =
+            new Object() {
```

Comment:
This doesn't test that the varargs constructor can be used to create an anonymous class. It should be something like `new VarargType(...) { /* either nothing or some code */ }`

Reply:
Okay, thanks!